components/server/src/api/prebuild-service-api.ts

```diff
@@ -120,7 +120,9 @@ export class PrebuildServiceAPI implements ServiceImpl<typeof PrebuildServiceInt
             });
             configurationId = resp.prebuild!.configurationId;
         }
-        const it = this.prebuildManager.watchPrebuildStatus(ctxUserId(), configurationId);
+        const it = await this.prebuildManager.watchPrebuildStatus(ctxUserId(), configurationId, {
```

Comment:
here we were using promise as async generator 😬  took me awhile to understand why it returns immediately with 200